src/_locales/es/messages.json

```diff
@@ -278,39 +278,39 @@
         }
     },
     "intro_beyond_ads": {
-        "message": "Atrapa rastreadores ocultos",
+        "message": "Descubre rastreadores ocultos",
```

Comment:
I would keep "Atrapa" or use an other synonymous since "Descubre" means only discover and we want to say it also catches them.

Reply:
@andresbase, now I think that "neutraliza" would be a more appropriate word. To me it implies that Badger finds and block the cookies. What do you think?

Reply:
Yes, I like neutraliza. Thanks!